src/prefect/blocks/core.py

```diff
@@ -409,11 +409,10 @@ def _parse_docstring(cls) -> List[DocstringSection]:
         `<module>:11: No type or annotation for parameter 'write_json'`
         because griffe is unable to parse the types from pydantic.BaseModel.
         """
-        griffe_logger = logging.getLogger("griffe.docstrings.google")
-        griffe_logger.disabled = True
-        docstring = Docstring(cls.__doc__)
-        parsed = parse(docstring, Parser.google)
-        griffe_logger.disabled = False
+        with disable_logger("griffe.docstrings.google"):
+            with disable_logger("griffe.agents.nodes"):
```

Comment:
Wanted to put this in one line, but too long on one line and it didn't like a parentheses:
https://github.com/PrefectHQ/prefect/pull/7059/commits/d4c3955ddd577c60ec812dbd55d05648a127c300